tests/event_discovery.tests/implicit.json

```diff
@@ -0,0 +1,37 @@
+//
+// Discovery result for a network scan (e.g. BACnet discovery scan), including
+// an implicit object point enumeration.
+//
+// Sent on MQTT topic: /devices/{device_id}/events/discovery
+// where {device_id} is that of the discovery node (e.g. gateway). The {device_id}
```

Comment:
The last sentence is confusing because it uses the same {device_id} - I assume it's to say that if a discovered device has a IoT ID, then this is provided in the families.iot.id field but the message is always published to the discovery node (/devices/{discovery node id}/events/discovery)?

Reply:
Yes -- it's tricky because it's some nuanced information that isn't going to be explained in a single comment. I've tried to make it a bit better, but I think here it just needs to be technically accurate, but it's not trying to explain everything.